### PR TITLE
Testing PR automation -- DO NOT MERGE

### PR DESCRIPTION
**Pull request by eiriktsarpalis**

### CODE REVIEW

**Comment by eiriktsarpalis** [comment]
Author feedback

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
```
Comment (trylek) [inline src/libraries/System.Text.Json/src/System.Text.Json.csproj:2]: Superfluous blank line, please remove. (Just to test the change request logic.)

**Review by trylek — DISMISSED** [review]

**Comment by eiriktsarpalis** [comment]
Author feedback

**Comment by eiriktsarpalis** [comment]
Author feedback.

**Comment by eiriktsarpalis** [comment]
Author feedback.

**Comment by eiriktsarpalis** [comment]
Author feedback.

**Comment by eiriktsarpalis** [comment]
Author comment.

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj (hunk 1 above)
Comment (joperezr) [inline src/libraries/System.Text.Json/src/System.Text.Json.csproj:2]: I don't think you have addressed this comment.

**Review by joperezr — CHANGES_REQUESTED** [review]
left a small comment. Please fix!

File: src/libraries/System.Text.Json/src/System.Text.Json.csproj (hunk 1 above)
Comment (eiriktsarpalis) [reply src/libraries/System.Text.Json/src/System.Text.Json.csproj:2]: Author response to review comment.

**Comment by eiriktsarpalis** [comment]
Author comment

**Pull request closed without merging**